File: Assets/AltUnityTester/Examples/Test/Editor/MockClasses/MockNotificationCallBacks.cs
using Altom.AltUnityDriver.Notifications;
namespace Altom.AltUnityTester.MockClasses
{
    internal class MockNotificationCallBacks : INotificationCallbacks
    {
        public static string LastSceneLoaded;
        public void SceneLoadedCallback(AltUnityLoadSceneNotificationResultParams altUnityLoadSceneNotificationResultParams)
        {
            LastSceneLoaded = altUnityLoadSceneNotificationResultParams.sceneName;
        }
    }
}

File: Bindings~/java/src/test/java/ro/altom/altunitytester/MockNotificationCallBacks.java
package ro.altom.altunitytester;

import ro.altom.altunitytester.Notifications.AltUnityLoadSceneNotificationResultParams;
import ro.altom.altunitytester.Notifications.BaseNotificationCallbacks;

public class MockNotificationCallBacks extends BaseNotificationCallbacks {

    public static String sceneName;
    public static String objectName;

    @Override
    public void SceneLoadedCallBack(
            AltUnityLoadSceneNotificationResultParams altUnityLoadSceneNotificationResultParams) {
        sceneName = altUnityLoadSceneNotificationResultParams.sceneName;
    }

    @Override
    public void HierarchyCallback(
            AltUnityHierarchyChangedNotificationResultParams altUnityHierarchyChangedNotificationResultParams) {
        objectName = altUnityHierarchyChangedNotificationResultParams.objectName;
    }
    

}
